runtime/frequency/src/lib.rs

```diff
@@ -286,7 +286,8 @@ impl Contains<RuntimeCall> for PasskeyCallFilter {
 
 			RuntimeCall::Balances(pallet_balances::Call::transfer_keep_alive { .. }) |
 			RuntimeCall::Balances(pallet_balances::Call::transfer_allow_death { .. }) |
-			RuntimeCall::Balances(pallet_balances::Call::transfer_all { .. }) => true,
+			RuntimeCall::Balances(pallet_balances::Call::transfer_all { .. }) |
```

Comment:
@aramikm shouldn't this be just this to allow for any type of token transfer?

```suggestion
			RuntimeCall::Balances(_)
```

Reply:
It is pretty much the same in practice since the rest of the extrinsics inside that pallet all requires root access. `ensure_root` 

I can modify it if you think that might be simpler.

Reply:
![image](https://github.com/user-attachments/assets/416b3b46-3df3-4cb3-a697-7f88d29f1fdc)
There are at least these other two that should be allowed right?

I think allowing all is simplest way

Reply:
But those are allowed if you look at the allowed list but I'll modify it to just include all.

```
			RuntimeCall::Balances(pallet_balances::Call::transfer_keep_alive { .. }) |
			RuntimeCall::Balances(pallet_balances::Call::transfer_allow_death { .. }) |
			RuntimeCall::Balances(pallet_balances::Call::transfer_all { .. }) |
			RuntimeCall::Capacity(_) => true,
```

Reply:
Ah I completely missed those up above. Sorry about that. Thanks for the simplification PR.